models/cyberconnect/bnb/cyberconnect_bnb_events.sql

```diff
@@ -0,0 +1,154 @@
+{{ config(
+    alias = 'events',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'name', 'evt_tx_hash', 'evt_index'],
```

Comment:
There may be multiple records with same unique_key in incremental updates:
https://github.com/duneanalytics/spellbook/actions/runs/4855417401/jobs/8653941058#step:12:26

We may need to either strengthen the unique_key or the distinct selection.

Reply:
The dupes may from the `Mint` part:
https://dune.com/queries/2432947?d=1